_data/pricing.yml

```diff
@@ -44,6 +44,13 @@ addons:
       label: "4<span class='small'>,</span>950"
       value: 4950
       monthly: false
+  cis_benchmark_compliance:
+    name: CIS Benchmark Compliance
+    details: per month
+    price:
+      label: "4<span class='small'>,</span>395"
+      value: 9950
+      monthly: false
```

Comment:
I'm confused... `details` says "per month"... This says `monthly: false`... The label is 4,395... The `value` is 9950... And the HTML (in another file below) says "Contact Us"?

Reply:
My mistake, a vestige of an earlier revision. I've simply removed mention of CIS pricing here instead preferring "Contact Us".